pom.xml

```diff
@@ -58,8 +58,8 @@
       <guava-version>31.1-jre</guava-version>
       <guice-version>5.1.0</guice-version>
       <hibernate-validator-version>8.0.0.Final</hibernate-validator-version>
-      <io-admin-shell-aas-model>1.2.0</io-admin-shell-aas-model>
-      <io-admin-shell-aas-serializer>1.2.1</io-admin-shell-aas-serializer>
+      <io-admin-shell-aas-model>1.0.0-milestone-03.1</io-admin-shell-aas-model>
+      <io-admin-shell-aas-serializer>1.0.0-milestone-03.1</io-admin-shell-aas-serializer>
```

Comment:
Better introduce a new set of properties and the proper dependency management for the AAS4J dependencies?

Reply:
I don't know the details of io-admin-shell, but isn't a version '1.0.0-milestone' older than a version '1.2.0'?

Reply:
> I don't know the details of io-admin-shell, but isn't a version '1.0.0-milestone' older than a version '1.2.0'?

Looks strange, right. However, they transitioned to Eclipse Foundation and reorganized the whole project, and because of that startet over again with version numbering.